src/contracts/core/DelegationManager.sol

```diff
@@ -173,42 +190,31 @@ contract DelegationManager is
         uint64[] memory maxMagnitudes = allocationManager.getMaxMagnitudes(operator, strategies);
 
         for (uint256 i = 0; i < strategies.length; i++) {
-            StakerScalingFactors storage ssf = stakerScalingFactor[staker][strategies[i]];
-
             // If the operator was not slashed 100% for the strategy and the staker has not been fully slashed
```

Comment:
Can we remove/update this comment? This is no longer true and we will have withdrawal roots produced even if the operator is 100% slashed. ScaledShares will just be 0 in this case

Reply:
[2ea120f](https://github.com/Layr-Labs/eigenlayer-contracts/pull/876/commits/2ea120f280126f1f2d32e8baad70e859ecea0175)